.travis.yml

```diff
@@ -3,6 +3,18 @@ language: ruby
 
 sudo: required
 
+# If the change is a markdown file, don't run CI build. Or at a later point
+# we can set a specific job in here (such as a markdown lint)
+before_install:
+- |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+      TRAVIS_COMMIT_RANGE="FETCH_HEAD...$TRAVIS_BRANCH"
+    fi
+    git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md$)|(^docs)/' || {
+      echo "Only markdown files were updated, stopping build process."
```

Comment:
Technically, I think this should be "Only documentation files", right?

Reply:
Agree!